src/render/blog/hello-rest-service.html.md

```diff
@@ -14,53 +14,408 @@ This article concentrates on integration with service using Knot.x, instead of c
-              
-## Book service – the service we’re going to integrate with             
+
+## Book service – the service we’re going to integrate with
 
 We’re going to use [Google Books API](https://developers.google.com/books/) as a service feed with books. 
 To make the tutorial easy, we use very simple endpoint that [search all volumes](https://developers.google.com/books/docs/v1/using#WorkingVolumes) for the given query.
 
-The goal is to display all volumes related to <strong>java</strong> term 
+The goal is to display all volumes related to **java** term 
 ([https://www.googleapis.com/books/v1/volumes?q=java](https://www.googleapis.com/books/v1/volumes?q=java)).
 Sample JSON result of the service looks like below:
 
 ``` json
 {
-   "kind":"books#volumes",
-   "totalItems":605,
-   "items":[
-      {
-         "kind":"books#volume",
-         "id":"87totx4p3ZcC",
-         "etag":"mviM5CD/Vg8",
-         "selfLink":"https://www.googleapis.com/books/v1/volumes/87totx4p3ZcC",
-         "volumeInfo":{
-            "title":"Java in a Time of Revolution",
-            "subtitle":"Occupation and Resistance, 1944-1946",
-            "authors":[
-               "Benedict Anderson"
-            ],
-            "publisher":"Equinox Publishing",
-            "publishedDate":"2005-12-01",
-            "description":"With remarkable scope and in scrupulous detail...",
-            "pageCount":494,
-            "imageLinks":{
-               "smallThumbnail":"...",
-               "thumbnail":",,,"
-            },
-            "language":"en"
-         }
-      },
-      ...
-   ]
+  "kind":"books#volumes",
+  "totalItems":605,
+  "items":[
+    {
+      "kind":"books#volume",
+      "id":"87totx4p3ZcC",
+      "etag":"mviM5CD/Vg8",
+      "selfLink":"https://www.googleapis.com/books/v1/volumes/87totx4p3ZcC",
+      "volumeInfo":{
+        "title":"Java in a Time of Revolution",
+        "subtitle":"Occupation and Resistance, 1944-1946",
+        "authors":[
+          "Benedict Anderson"
+        ],
+        "publisher":"Equinox Publishing",
+        "publishedDate":"2005-12-01",
+        "description":"With remarkable scope and in scrupulous detail...",
+        "pageCount":494,
+        "imageLinks":{
+          "smallThumbnail":"...",
+           "thumbnail":"..."
+         },
+         "language":"en"
+       }
+    },
+    ...
+  ]
+}
+```
+
+## Our application
+Knot.x provides the functionality that combines service data with HTML in a very efficient manner. 
+It can get a JSON response from a service and then using Handlebars engine render the dynamic part. 
+This can be achieved without any custom code – it is only the matter of configuration.
+
+The diagram below depicts how a request from a user is handled with the Knot.x application.
+
+<img src="/img/blog/hello-rest-service/books-service-diagram.jpg" class="img-responsive img-centered" alt="Books service diagram"/>
```

Comment:
Shouldn't you use markdown to display image ? If it's not nicely rendered let me know - as the CSS tweaks need to be done.